commit/merkleroot/rmn/config.go

```diff
@@ -33,7 +34,8 @@ type RMNNodeInfo struct {
 	IsSigner                  bool
 	SignReportsAddress        cciptypes.Bytes
 	SignObservationsPublicKey *ed25519.PublicKey // offChainPublicKey
-	SignObservationPrefix     string             // e.g. "chainlink ccip 1.6 rmn observation"
+	// TODO: clarify this field
+	SignObservationPrefix string // e.g. "chainlink ccip 1.6 rmn observation"
```

Comment:
@dimkouv this field is not fetched by the `GetRMNRemoteCfg`. Is it something offline?

Reply:
This feels like something that might need to be in either the Commit offchain config or the shared config b/w RMN and CCIP.

Reply:
I see. Makes sense I was confused if we considered it to be in the contracts. I could maybe have it as part of this [PR](https://github.com/smartcontractkit/chainlink-ccip/pull/195) where I am removing the CommonPluginConfig in favor of the Offchain config.